src/main/java/com/openlattice/chronicle/services/ChronicleServiceImpl.java

```diff
@@ -595,7 +610,7 @@ public Integer updateAppsUsageAssociationData(
 
         SearchApi searchApi;
         try {
-            ApiClient apiClient = apiClientCache.get( ApiClient.class );
+            ApiClient apiClient = intApiClientCache.get( ApiClient.class );
```

Comment:
I think this one should be `prodApiClientCache` since it's used for getting data for the survey.